### PR TITLE
use x-api-key request header

### PR DESCRIPTION
**Pull request by StephenHodgson**

### CODE REVIEW

File: Runtime/BlockadeLabsSkyboxGenerator.cs [hunk 1]
```diff
@@ -25,8 +25,7 @@ public class BlockadeLabsSkyboxGenerator : MonoBehaviour
         private string _apiKey = "API key needed. Get one at api.blockadelabs.com";
         public string ApiKey
         {
-            get => _apiKey;
-            set => _apiKey = value;
+            get => ApiRequests.ApiKey;
```
Comment (afarchy) [inline Runtime/BlockadeLabsSkyboxGenerator.cs:28]: revert - this return the wrong value until CheckApiKeyValid is checked

File: Runtime/BlockadeLabsSkyboxGenerator.cs [hunk 2]
```diff
@@ -534,13 +534,13 @@ private async Task<GetImagineResult> PollForResultAsync(string imagineObfuscated
 
         private async Task WaitForSeconds(float seconds)
         {
-            #if UNITY_EDITOR
-                await Task.Delay((int)(seconds * 1000)).ConfigureAwait(true);
-            #else
+#if UNITY_EDITOR
+            await Task.Delay((int)(seconds * 1000)).ConfigureAwait(true);
```
Comment (afarchy) [inline Runtime/BlockadeLabsSkyboxGenerator.cs:538]: tabbing error

File: Runtime/BlockadeLabsSkyboxGenerator.cs (hunk 1 above)
Comment (afarchy) [reply Runtime/BlockadeLabsSkyboxGenerator.cs:28]: and I think users will need the setter if they're using this through code?

**Review by afarchy — CHANGES_REQUESTED** [review]

File: Runtime/BlockadeLabsSkyboxGenerator.cs (hunk 2 above)
Comment (afarchy) [reply Runtime/BlockadeLabsSkyboxGenerator.cs:544]: I guess the tabbing error is in the else case? It's wrong somewhere.

**Review by afarchy — APPROVED** [review]

File: Runtime/BlockadeLabsSkyboxGenerator.cs (hunk 2 above)
Comment (StephenHodgson) [reply Runtime/BlockadeLabsSkyboxGenerator.cs:547]: image

Looks right to me. idk. let me know

Comment (StephenHodgson) [reply Runtime/BlockadeLabsSkyboxGenerator.cs:547]: I see it now

**Pull request merged**